Formula/ballerina.rb

```diff
@@ -21,17 +24,26 @@ def install
       s.gsub! /\r?/, ""
     end
 
+    inreplace ["bin/composer"] do |s|
+      # Translate composer script
+      s.gsub! /^BASE_DIR=.*$/, "BASE_DIR=#{libexec}/bin"
+      s.gsub! /^PRGDIR=.*$/, "PRGDIR=#{libexec}/bin"
+      # dos to unix (bug fix for version 2.3.11)
```

Comment:
What this issue reported? And what is “version 2.3.11”?

Reply:
Borrowed it from https://github.com/Linuxbrew/homebrew-core/blob/master/Formula/akka.rb.  Seems version is not related here.